### PR TITLE
systemd: Use systemd-sysusers to create cockpit-wsinstance user

### PR DESCRIPTION
**Pull request by martinpitt**
This was originally proposed and discussed extensively in @travier 's PR #20365. However, all of a sudden  I'm not allowed to push into @travier's branch any more:
```
To github.com:travier/cockpit.git
 ! [remote rejected]     HEAD -> main-sysuserconfig (permission denied)
error: failed to push some refs to 'github.com:travier/cockpit.git'
```

So cloning the PR from my fork.

### CODE REVIEW

**Comment by martinpitt** [comment]
[These](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20447-84385379-20240508-053530-fedora-coreos-networking/log.html) [are](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20447-84385379-20240508-053533-rhel4edge-networking/log.html#14-2) a problem. I'll back out the tmpfiles.d commit for now -- even without it it's already an improvement, and I'll let the tmpfiles.d thing simmer for a bit longer.

File: src/systemd/sysusers.d/cockpit-ws.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+u cockpit-wsinstance - "User for cockpit-ws instances" -
```
Comment (allisonkarlitskaya) [inline src/systemd/sysusers.d/cockpit-ws.conf:1]: Why is the filename different from the username?  That seems strange to me...

**Review by allisonkarlitskaya — APPROVED** [review]
This looks good except for one trivial nag.  Please either merge as-is, do a force-push, or I'm also fine with a follow-up (ideally before release).

Thanks!

File: src/systemd/sysusers.d/cockpit-ws.conf (hunk 1 above)
Comment (martinpitt) [reply src/systemd/sysusers.d/cockpit-ws.conf]: Indeed, that's fallout from the original PR version having both users, but we eliminated cockpit-ws in the meantime. I renamed it.

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**